---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
library(ggplot2)
library(RColorBrewer)
library(wordcloud)
library(here)
library(scholar)
library(magrittr)
library(dplyr)
library(glue)
library(stringr)
```

<!-- badges: start -->
<!-- badges: end -->

```{r cache=FALSE, echo=FALSE}
knitr::read_chunk('functions.R')
```

```{r plot_bridge, echo=FALSE}
```

```{r showbridge, echo = FALSE, fig.height=1, fig.width=4, fig.align='center'}
plot_bridge(2)
```


<!-- ## Passionate about optimising how humans work with data -->
<!-- ## Passionate about bridging the gap between humans and their data -->

## A passion for building bridges between people and their data

- Andrea Marion Marquard
- 23 Feb 1986
- Copenhagen, Denmark

I have an eye for optimisation and automation of data analysis workflows, and I believe in the power of giving people meaningful and hassle-free access to their data.

This means creating interactive data dashboards, and automating reporting by creating eg. Word/Excel/Powerpoint products directly from R.

To bring these solutions to life I use R package development, R markdown, Shiny, tidyverse, unit testing, github actions for CI/CD and bash where needed.

I want to make life easier for both the data scientist as well as the people who own the data.

About this CV:

- it lives at: https://github.com/amarquard/cv
- it is automatically rendered from an R markdown file whenever there is a new commit to the repo. Check out the [source code](README.Rmd)
- the publications list always up to date. It is created by a direct call to Google Scholar
- scroll to the bottom to see how the bridge above was built

### Experience

`Jun 2021 - Mar 2022` **Maternity leave** (9 months)

`Mar 2021 - Present` **Director, Head of Data Science & Automation** (Clinical Microbiomics)

* Leading efforts to automate and scale the data analysis operations
* Lead developer of internal R packages and Shiny dashboards
* Managing a team of student assistants

`Aug 2019 - Mar 2021` **Bioinformatics specialist** (Clinical Microbiomics)

* Statistical analysis of microbiome data, and reporting results to clients
* Identified a need for better code sharing, automation and reproducibility
* Initiated and led development of:
  - internal R package to track and maintain analysis code
  - internal R package to automate generation of reports
  - code testing during CI, and branch protection
  - Shiny dashboards for clients to interact with their data

`Jun 2019` **Freelance bioinformatics expert** (Immumap Services)

* Developed an NGS software tool with a Shiny interface to be used by lab staff

`Aug 2017 - Aug 2018` **Maternity leave** (12 months)

`Oct 2016 - May 2019` **Postdoc** (T-cells and Cancer lab, DTU Health Technology)

* Data analysis pipeline for single cell sequencing of T cells
* Custom analysis pipelines for synthetic DNA barcodes
* Pipeline to predict neoepitopes at exon-exon junctions in aberrant mRNA splicing
* R package for predicting cross-reactivity of T-cells for immunotherapy
* Shiny dashboard for ggplot2-powered visualisation of high-throughput data (available [here](https://marquard.shinyapps.io/barracoda-viz/))

`Jul 2015 - Apr 2016` **Maternity leave** (9 months)
  
`Jun 2015` **External research stay** (Ton Schumacher Lab, Antoni van Leeuwenhoek Hospital, Amsterdam)


### Awards

`2017` Recipient of DTU’s Young Researcher Award


### Education

`2012 - 2016` **PhD in Cancer Genomics** (DTU Bioinformatics)

  * A tool for analysing NGS data from a novel screening method using DNA barcodes
    (available [here](https://services.healthtech.dtu.dk/service.php?Barracoda-1.8))
  * R package for machine learning-based tracing of tissue source of cancer 
    cells (interface available [here](https://services.healthtech.dtu.dk/service.php?TumorTracer-1.1))
  * Mutation patterns in genomic data from >5000 cancer patients reveal vulnerability to chemotherapy
  
  
`2012` **Machine Learning course** (by Stanford Uni at Coursera.org)  

`2006 - 2012` **BSc and MSc in Molecular Biomedicine** (Uni. of Copenhagen)

`2006` **Study abroad** (University of New South Wales, Australia)


### Volunteering & student jobs

`2007 - 2011` **Board member and chairman** (Biocenter Student House) | *Learning to organise events, people and paperwork.*

`2010 - 2011` **Student assistant** (The Finsen Laboratory, Copenhagen University Hospital) | *Responsible for genotyping of mice for breeding and experiments.*

`2008 – 2010` **Student assistant** (Pediatric Oncology Research Laboratory, Copenhagen University Hospital) | *Preparation of patient blood samples for diagnosis and research, eg. buffy coat and DNA extraction.*

`2007 – 2008` **Assistant press officer** (Dept. of Biology, University of Copenhagen) | *Design of department website and English translations.*



### Publications
 

```{r scholar_functions, echo=FALSE}
```

```{r, echo = FALSE}
# My unique google scholar id
id <- "UFlhrz0AAAAJ"

pubs <- scholar::get_publications(id)

pubs %>%
  # Removing conference abstracts (no citations) and patent applications (no journal)
  filter(cites > 0 & journal != "") %>%
  mutate(
    # Format my name in bold
    author = format_myself(author, "AM Marquard"),
    # Create a link to paper in google scholar, and add to the title
    link = make_scholar_link(pubid, .env$id),
    title = str_trunc(title, 50),
    title = glue("[{title}]({link})")) %>% 
  select(-cid, -pubid, -link) %>%
  arrange(desc(year), desc(cites)) %>%
  knitr::kable()
```


### How I built the bridge

```{r plot_bridge, eval=FALSE}
```

