---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
library(ggplot2)
library(RColorBrewer)
library(wordcloud)
library(here)
library(scholar)
library(magrittr)
library(dplyr)
```

<!-- badges: start -->
<!-- badges: end -->

```{r cache=FALSE, echo=FALSE}
knitr::read_chunk('functions.R')
```

```{r plot_bridge, echo=FALSE}
```

```{r showbridge, echo = FALSE, fig.height=1, fig.width=4, fig.align='center'}
plot_bridge(2)
```


## The biologist who learnt to code 
I build bridges between wet-lab scientists and bioinformaticians. I've spent >5 years on either side of the divide and I therefore speak both lab-talk and code-lingo. Below is an overview of what I do: I understand this <span style="color:#238B45;font-weight:bold;font-size:larger">biology</span>, I can study it with these <span style="color:#CB181D;font-weight:bold;font-size:larger">technologies</span>, and I can make sense of the data using these <span style="color:#2171B5;font-weight:bold;font-size:larger">computer programs</span>.
<!-- Expand a little, perhaps tailored to Novo -->

```{r word_cloud, echo=FALSE}
```

```{r echo = FALSE, fig.height = 3, fig.width = 7, fig.asp = .33, fig.align='center'}
source(here("words-for-clouds.R"))
par(mfrow=c(1,3))
word_cloud(bio, brewer.pal(8, "Greens"), 11)
word_cloud(tech, brewer.pal(8, "Reds"), 12)
word_cloud(comp, brewer.pal(8, "Blues"), 6)
```


### Experience

`Jun 2021 - Apr 2022` **Maternity leave** (10 months)

`Mar 2021 - Present` **Director, Head of Data Science & Automation** (Clinical Microbiomics)

`Aug 2019 - Mar 2021` **Bioinformatics specialist** (Clinical Microbiomics)

* 

`Jun 2019` **Freelance bioinformatics expert** (Immumap Services)

* I developed a software tool needed for analysis of NGS data from the dCODE 
  platform used for detection of more than 1000 different antigen-specific
  T cells. 

`Aug 2017 - Aug 2018` **Maternity leave** (12 months)

`Oct 2016 - May 2019` **Postdoc** (Hadrup Lab, DTU Health Technology)

* The group works on immunotherapy of cancer and autoimmune diseases and 
  consists of 30 experimentalists and 2 bioinformaticians. I completed the
  following projects:
    1. Establish an analysis pipeline for single cell sequencing of T cells
       in parallel with antigen-specificity screening using DNA barcodes
    2. Predict neoepitopes arising at novel exon-exon junctions upon aberrant
       mRNA splicing in disease (such as myelodysplastic syndrome)
    3. Predicting cross-reactivity of T cells to avoid adverse effects of 
       immunotherapy *(published)* 
    4. Prediction of antigenic peptides in mouse tumor models *(published)*
* My tasks also included data analysis and visualisation for other projects, 
  as well as setting up automated analysis where possible to minimize errors 
  and workload

`Jul 2015 - Apr 2016` **Maternity leave** (9 months)
  
`Jun 2015` **External research stay** (Ton Schumacher Lab, Antoni van Leeuwenhoek Hospital, Amsterdam)

* AI-based prediction of antigenic peptides for cancer immunotherapy


<!-- #### Education -->

<!-- * `2012 - 2016`  PhD in Cancer Genomics group, DTU Bioinformatics -->
<!--     + *Title: Therapeutic applications of the cancer genome* -->
<!--     + *I used machine learning to trace the organ source of cancer cells via DNA analyses [Marquard et al, BMC Medical Genomics 2015]* -->
<!--     + *I measured patterns of mutations in >5000 cancer patients, to reveal vulnerability to certain types of chemotherapy [Marquard et al, Biomarker Research 2015]* -->
<!--     + *I made a user-friendly program for analysing sequencing results of a novel screening method using DNA barcodes [Bentzen et al, Nature Biotehcnology 2016]* -->
<!--     + *courses include: perl and unix for bioinformaticians, tumor biology, systems biology* -->

<!-- * `2012` Machine Learning course by Andrew Ng at Stanford Uni (on Coursera.org)   -->
<!--     + *completed all exercises and tests* -->
<!--     + *implemented (from scratch) a neural network with backpropagation to image-recognition of hand-written digits* -->

<!-- * `2006 - 2012` BSc and MSc in Molecular Biomedicine, Uni. of Copenhagen -->
<!--     + *2 yrs lab-work for Master thesis in malaria lab* -->
<!--     + *Hands-on experience with: cell cultures, FACS, ELISA, qPCR, Biacore * -->
<!--     + *FELASA-C animal certificate* -->
<!--     + *courses include: molecular and cell biology, genetics, pathology, biochemistry, statistics, cell signaling, immunology* -->

<!-- * `2006` Study abroad at University of New South Wales, Australia -->

### Awards

`2017` Recipient of DTU’s Young Researcher Award


### Education

`2012 - 2016` **PhD in Cancer Genomics** (DTU Bioinformatics)

* Title: Therapeutic applications of the cancer genome. Incl. 3 published papers:
    1. I used machine learning to trace the organ source of cancer cells via DNA analyses
    2. I measured patterns of mutations in >5000 cancer patients, to reveal vulnerability to certain types of chemotherapy
    3. I made a user-friendly program for analysing sequencing results of a novel screening method using DNA barcodes
* Courses include:
    + perl and unix | biological sequence analysis
    + tumor biology |  computational network biology

  
`2012` **Machine Learning course** (by Stanford Uni at Coursera.org)  

* implemented (from scratch) a neural network with backpropagation for image-recognition of hand-written digits

`2006 - 2012` **BSc and MSc in Molecular Biomedicine** (Uni. of Copenhagen)

* 2 yrs in malaria lab for MSc thesis, including hands-on: cell cultures, FACS, ELISA, qPCR, Biacore
* FELASA-C laboratory animal science certificate
* Courses include:
    + molecular and cell biology | cell signaling | pathology
    + protein- and biochemistry | immunology | physiology
    + genetics | statistics | bioinformatics

`2006` **Study abroad** (University of New South Wales, Australia)

* I studied biology, mathematics and databases, and was active in student societies.


### Workstyle

* I am structured in my work and have an attention for detail
* I would never assemble IKEA furniture without reading the manual thoroughly, and I therefore get it right the first time
* I am creative and innovative. This resume was made with R Markdown, and includes a hyperbolic cosine graph to look like a bridge


### Volunteering & student jobs

`2007 - 2011` **Board member and chairman** (Biocenter Student House) | *Learning to organise events, people and paperwork.*

`2010 - 2011` **Student assistant** (The Finsen Laboratory, Copenhagen University Hospital) | *Responsible for genotyping of mice for breeding and experiments.*

`2008 – 2010` **Student assistant** (Pediatric Oncology Research Laboratory, Copenhagen University Hospital) | *Preparation of patient blood samples for diagnosis and research, eg. buffy coat and DNA extraction.*

`2007 – 2008` **Assistant press officer** (Dept. of Biology, University of Copenhagen) | *Design of department website and English translations.*



### Publications

```{r, echo = FALSE}
id <- "UFlhrz0AAAAJ"
pubs <- scholar::get_publications(id)

apply_bold <- function(x) {
   paste0("**", x, "**")
}

format_myself <- function(x, me) {
  stringr::str_replace_all(x, me, apply_bold(me))
}

pubs %>%
  filter(cites > 0) %>%
  mutate(author = format_myself(author, "AM Marquard")) %>% 
  select(-cid, -pubid) %>%
  arrange(year, cites) %>%
  knitr::kable()
```


### How I built the bridge:

```{r plot_bridge, eval=FALSE}
```

