
<!-- README.md is generated from README.Rmd. Please edit that file -->

<!-- badges: start -->

<!-- badges: end -->

<img src="README_files/figure-gfm/showbridge-1.png" style="display: block; margin: auto;" />

<!-- ## Passionate about optimising how humans work with data -->

<!-- ## Passionate about bridging the gap between humans and their data -->

## A passion for building bridges between people and their data

My passion is optimisation and automation 🤖 of data analysis workflows 🎡

My tools include R 💙, bash, perl 👾

My R world includes R package development 📦, R markdown, Shiny ✨,
tidyverse 🧹, unit testing

My strong suit is communicating with non-data people, through
interactive data dashboards 📊 🖥 and automated reporting by creating
Word/Excel/Powerpoint 📑 🎥 📈 directly from R

My workflow includes git version control 🐙 and continuous integration
(CI)

My domain knowledge includes immunotherapy 🧫, cancer 💊, microbiome 🦠,
infectious diseases 😷

### Experience

`Jun 2021 - Apr 2022` **Maternity leave** (10 months)

`Mar 2021 - Present` **Director, Head of Data Science & Automation**
(Clinical Microbiomics)

  - Leading efforts to automate and scale the data analysis operations
  - Lead developer of internal R packages and Shiny dashboards
  - Managing a team of student assistants

`Aug 2019 - Mar 2021` **Bioinformatics specialist** (Clinical
Microbiomics)

  - Analysed microbiome data and reported results to clients
  - Identified a need for better code sharing, automation and
    reproducibility
  - Independently initiated and led development of:
      - internal R package to track and maintain analysis code
      - internal R package to automate generation of reports
      - code testing during CI, and branch protection
      - Shiny dashboards for clients to interact with their data and
        statistical analyses

`Jun 2019` **Freelance bioinformatics expert** (Immumap Services)

  - Developed a software tool with a Shiny interface to be used by lab
    staff
      - The tool analyses NGS data from the dCODE platform, a
        single-cell approach to detect \>1000 antigen specificities of T
        cells in blood samples.

`Aug 2017 - Aug 2018` **Maternity leave** (12 months)

`Oct 2016 - May 2019` **Postdoc** (T-cells and Cancer lab, DTU Health
Technology)

  - Major accomplishments:
      - data analysis pipeline for single cell sequencing of T cells
        with antigen-specificity screening using DNA barcodes
      - pipeline to predict neoepitopes arising at novel exon-exon
        junctions upon aberrant mRNA splicing in disease
      - R package for predicting cross-reactivity of T cells to avoid
        adverse effects of immunotherapy *(published)*
      - Shiny dashboard for point-and-click ggplot-powered visualisation
        of high-throughput data (to replace Graph Pad Prism)
  - I supported the research team with *ad hoc* data analysis and
    visualisation
  - I set up automated analyses where possible to minimize human errors
    and workload

`Jul 2015 - Apr 2016` **Maternity leave** (9 months)

`Jun 2015` **External research stay** (Ton Schumacher Lab, Antoni van
Leeuwenhoek Hospital, Amsterdam)

### Awards

`2017` Recipient of DTU’s Young Researcher Award

### Education

`2012 - 2016` **PhD in Cancer Genomics** (DTU Bioinformatics)

  - Developed a user-friendly tool for analysing sequencing results of a
    novel screening method using DNA barcodes (available
    [here](https://services.healthtech.dtu.dk/service.php?Barracoda-1.8))
  - R package that uses machine learning to trace the organ source of
    cancer cells via DNA analyses (interface available
    [here](https://services.healthtech.dtu.dk/service.php?TumorTracer-1.1))
  - Infer patterns of mutations from genomic data in \>5000 cancer
    patients, to reveal vulnerability to certain types of chemotherapy

`2012` **Machine Learning course** (by Stanford Uni at Coursera.org)

`2006 - 2012` **BSc and MSc in Molecular Biomedicine** (Uni. of
Copenhagen)

`2006` **Study abroad** (University of New South Wales, Australia)

### Volunteering & student jobs

`2007 - 2011` **Board member and chairman** (Biocenter Student House) |
*Learning to organise events, people and paperwork.*

`2010 - 2011` **Student assistant** (The Finsen Laboratory, Copenhagen
University Hospital) | *Responsible for genotyping of mice for breeding
and experiments.*

`2008 – 2010` **Student assistant** (Pediatric Oncology Research
Laboratory, Copenhagen University Hospital) | *Preparation of patient
blood samples for diagnosis and research, eg. buffy coat and DNA
extraction.*

`2007 – 2008` **Assistant press officer** (Dept. of Biology, University
of Copenhagen) | *Design of department website and English
translations.*

### Publications

| title                                                                                                                                                                              | author                                                                  | journal                                         | number                | cites | year |
| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------- | :---------------------------------------------- | :-------------------- | ----: | ---: |
| [Human endogenous retroviruses form a reservoir …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:MXK_kJrjxJIC) | SK Saini, AD Ørskov, AM Bjerregaard, A Unnikrishnan, …                  | Nature communications                           | 11 (1), 1-14          |     8 | 2020 |
| [Tumor-infiltrating T cells from clear cell rena…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:0EnyYjriUFMC) | UK Hansen, S Ramskov, AM Bjerregaard, A Borch, R Andersen, A Draghi, …  | Frontiers in immunology                         | 11, 373               |     8 | 2020 |
| [CD8+ T cells from patients with narcolepsy and …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:UebtZRa9Y70C) | NW Pedersen, A Holm, NP Kristensen, AM Bjerregaard, AK Bentzen, …       | Nature communications                           | 10 (1), 1-12          |    47 | 2019 |
| [Prediction of neoepitopes from murine sequencin…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:LkGwnXOMwfcC) | AM Bjerregaard, TK Pedersen, **AM Marquard**, SR Hadrup                 | Cancer Immunology, Immunotherapy                | 68 (1), 159-161       |     3 | 2019 |
| [T cell receptor fingerprinting enables in-depth…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:roLk4NBRz8UC) | AK Bentzen, L Such, KK Jensen, **AM Marquard**, LE Jessen, NJ Miller, … | Nature biotechnology                            | 36 (12), 1191-1196    |    60 | 2018 |
| [A meta-proteomics approach to study the intersp…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:eQOLeE2rZwMC) | J Herschend, ZBV Damholt, **AM Marquard**, B Svensson, SJ Sørensen, …   | Scientific reports                              | 7 (1), 1-13           |    28 | 2017 |
| [Quantification of within-sample genetic heterog…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:IjCSPb-OGe4C) | P Martinez, C Kimberley, NJ BirkBak, A Marquard, Z Szallasi, TA Graham  | Scientific reports                              | 7 (1), 1-12           |     4 | 2017 |
| [Large-scale detection of antigen-specific T cel…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:qjMakFHDy7sC) | AK Bentzen, **AM Marquard**, R Lyngaa, SK Saini, S Ramskov, M Donia, …  | Nature biotechnology                            | 34 (10), 1037-1045    |   198 | 2016 |
| [Sequenza: allele-specific copy number and mutat…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:u5HHmVD_uO8C) | F Favero, T Joshi, **AM Marquard**, NJ Birkbak, M Krzystanek, Q Li, …   | Annals of Oncology                              | 26 (1), 64-70         |   368 | 2015 |
| [Pan-cancer analysis of genomic scar signatures …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:u-x6o8ySG0sC) | **AM Marquard**, AC Eklund, T Joshi, M Krzystanek, F Favero, ZC Wang, … | Biomarker research                              | 3 (1), 1-10           |   120 | 2015 |
| [TumorTracer: a method to identify the tissue of…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:9yKSN-GCB0IC) | **AM Marquard**, NJ Birkbak, CE Thomas, F Favero, M Krzystanek, …       | BMC medical genomics                            | 8 (1), 1-13           |    42 | 2015 |
| [Plasmodium falciparum expressing domain cassett…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:Tyk-4Ss8FVUC) | SS Berger, L Turner, CW Wang, JEV Petersen, M Kraft, JPA Lusingu, …     | PloS one                                        | 8 (7), e69117         |    57 | 2013 |
| [Plasmodium falciparum erythrocyte membrane prot…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:zYLM7Y9cAGgC) | T Lavstsen, L Turner, F Saguti, P Magistrado, TS Rask, JS Jespersen, …  | Proceedings of the National Academy of Sciences | 109 (26), E1791-E1800 |   258 | 2012 |
| [Evidence for in vitro and in vivo expression of…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:Y0pCki6q_DkC) | CW Wang, T Lavstsen, DC Bengtsson, PA Magistrado, SS Berger, …          | Malaria journal                                 | 11 (1), 1-11          |    28 | 2012 |

### How I built the bridge

``` r
#' Plot a bridge using cosine function and ggplot lines and segments
#'
#' @param p the x-axis value at which to plot the pillars (at p and -p) in the graph coordinates
#'
#' @return a ggplot object
#' @export
#'
#' @examples
#' plot_bridge(2)
#' 
plot_bridge <- function(p) {
  
  # x values around two pillars at position p and minus p
  x <- seq(-2*p, 2*p, length.out = 10000)
  
  # Hyperbolic cosine to get the centre curve
  y <- cosh(x)
  
  # Extend by repeating parts of the centre curve at the ends
  y[x < -p] <- y[x >= 0 & x <  p]
  y[x >  p] <- y[x <= 0 & x > -p]
  
  df <- data.frame(x = x, y = y)
  
  ggplot(df, aes(x = x, y = y)) +
    geom_line(size = 1) +
    ylim(0.5, cosh(p)) +
    
    # add road
    geom_hline(yintercept = 1, size = 1) +
    
    # add pillars
    geom_vline(xintercept = c(-p, p), size = 1.3) +
    
    # add ropes
    geom_segment(aes(x = x, y = 1, xend = x, yend = y),
                 data = df[round(seq(1, 10000, length.out = 61)),]) +
    
    # remove all axes etc.
    theme_void()
}
```
