---
output: github_document
---

<!-- README.md is generated from README.Rmd. Please edit that file -->

```{r, include = FALSE}
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>"
)
library(ggplot2)
library(RColorBrewer)
library(wordcloud)
library(here)
library(scholar)
library(magrittr)
library(dplyr)
library(glue)
library(stringr)
```

<!-- badges: start -->
<!-- badges: end -->

```{r cache=FALSE, echo=FALSE}
knitr::read_chunk('functions.R')
```

```{r plot_bridge, echo=FALSE}
```

```{r showbridge, echo = FALSE, fig.height=1, fig.width=4, fig.align='center'}
plot_bridge(2)
```


## The biologist who learnt to code 
I build bridges between wet-lab scientists and bioinformaticians. I've spent >5 years on either side of the divide and I therefore speak both lab-talk and code-lingo. 

### Experience

`Jun 2021 - Apr 2022` **Maternity leave** (10 months)

`Mar 2021 - Present` **Director, Head of Data Science & Automation** (Clinical Microbiomics)

* I am leading efforts to automate and scale the data analysis operations
* Managing a team of student assistants

`Aug 2019 - Mar 2021` **Bioinformatics specialist** (Clinical Microbiomics)

* Analysed microbiome data and reported results to clients
* Identified a need for better code sharing, automation and reproducibility
* Independently initiated and led development of:
  - an internal R package to track and maintain analysis code
  - an internal R package to automate generation of reports
  - code testing during CI, and branch protection
  - Shiny dashboards for clients to interact with their data and statistical analyses

`Jun 2019` **Freelance bioinformatics expert** (Immumap Services)

* I developed a software tool with a Shiny interface to be used by lab staff. The tool analyses NGS data from the dCODE platform, a single-cell approach to detect >1000 antigen specificities of T cells in blood samples. 

`Aug 2017 - Aug 2018` **Maternity leave** (12 months)

`Oct 2016 - May 2019` **Postdoc** (T-cells and Cancer lab, DTU Health Technology)

* Major accomplishments:
    - data analysis pipeline for single cell sequencing of T cells with antigen-specificity screening using DNA barcodes
    - pipeline to predict neoepitopes arising at novel exon-exon junctions upon aberrant mRNA splicing in disease
    - R package for predicting cross-reactivity of T cells to avoid adverse effects of 
       immunotherapy *(published)* 
    - Shiny dashboard for point-and-click ggplot-powered visualisation of high-throughput data (to replace Graph Pad Prism)
* I supported the research team with ad hoc data analysis and visualisation
* I setup automated analysis where possible to minimize human errors 
  and workload

`Jul 2015 - Apr 2016` **Maternity leave** (9 months)
  
`Jun 2015` **External research stay** (Ton Schumacher Lab, Antoni van Leeuwenhoek Hospital, Amsterdam)

* AI-based prediction of antigenic peptides for cancer immunotherapy


<!-- #### Education -->

<!-- * `2012 - 2016`  PhD in Cancer Genomics group, DTU Bioinformatics -->
<!--     + *Title: Therapeutic applications of the cancer genome* -->
<!--     + *I used machine learning to trace the organ source of cancer cells via DNA analyses [Marquard et al, BMC Medical Genomics 2015]* -->
<!--     + *I measured patterns of mutations in >5000 cancer patients, to reveal vulnerability to certain types of chemotherapy [Marquard et al, Biomarker Research 2015]* -->
<!--     + *I made a user-friendly program for analysing sequencing results of a novel screening method using DNA barcodes [Bentzen et al, Nature Biotehcnology 2016]* -->
<!--     + *courses include: perl and unix for bioinformaticians, tumor biology, systems biology* -->

<!-- * `2012` Machine Learning course by Andrew Ng at Stanford Uni (on Coursera.org)   -->
<!--     + *completed all exercises and tests* -->
<!--     + *implemented (from scratch) a neural network with backpropagation to image-recognition of hand-written digits* -->

<!-- * `2006 - 2012` BSc and MSc in Molecular Biomedicine, Uni. of Copenhagen -->
<!--     + *2 yrs lab-work for Master thesis in malaria lab* -->
<!--     + *Hands-on experience with: cell cultures, FACS, ELISA, qPCR, Biacore * -->
<!--     + *FELASA-C animal certificate* -->
<!--     + *courses include: molecular and cell biology, genetics, pathology, biochemistry, statistics, cell signaling, immunology* -->

<!-- * `2006` Study abroad at University of New South Wales, Australia -->

### Awards

`2017` Recipient of DTU’s Young Researcher Award


### Education

`2012 - 2016` **PhD in Cancer Genomics** (DTU Bioinformatics)

* Title: Therapeutic applications of the cancer genome. Incl. 3 published papers:
    1. I used machine learning to trace the organ source of cancer cells via DNA analyses
    2. I measured patterns of mutations in >5000 cancer patients, to reveal vulnerability to certain types of chemotherapy
    3. I made a user-friendly tool for analysing sequencing results of a novel screening method using DNA barcodes (available [here](https://services.healthtech.dtu.dk/service.php?Barracoda-1.8))

  
`2012` **Machine Learning course** (by Stanford Uni at Coursera.org)  

* implemented (from scratch) a neural network with backpropagation for image-recognition of hand-written digits

`2006 - 2012` **BSc and MSc in Molecular Biomedicine** (Uni. of Copenhagen)

`2006` **Study abroad** (University of New South Wales, Australia)


### Volunteering & student jobs

`2007 - 2011` **Board member and chairman** (Biocenter Student House) | *Learning to organise events, people and paperwork.*

`2010 - 2011` **Student assistant** (The Finsen Laboratory, Copenhagen University Hospital) | *Responsible for genotyping of mice for breeding and experiments.*

`2008 – 2010` **Student assistant** (Pediatric Oncology Research Laboratory, Copenhagen University Hospital) | *Preparation of patient blood samples for diagnosis and research, eg. buffy coat and DNA extraction.*

`2007 – 2008` **Assistant press officer** (Dept. of Biology, University of Copenhagen) | *Design of department website and English translations.*



### Publications

```{r scholar_functions, echo=FALSE}
```

```{r, echo = FALSE}
id <- "UFlhrz0AAAAJ"
pubs <- scholar::get_publications(id)

apply_bold <- function(x) {
   paste0("**", x, "**")
}

format_myself <- function(x, me) {
  stringr::str_replace_all(x, me, apply_bold(me))
}

pubs %>%
  filter(cites > 0 & journal != "") %>%
  mutate(author = format_myself(author, "AM Marquard")) %>% 
  mutate(link = make_scholar_link(pubid, .env$id)) %>% 
  mutate(title = str_trunc(title, 50)) %>% 
  mutate(title = glue("[{title}]({link})")) %>% 
  select(-cid, -pubid, -link) %>%
  arrange(desc(year), desc(cites)) %>%
  knitr::kable()
```


### How I built the bridge:

```{r plot_bridge, eval=FALSE}
```

