
<!-- README.md is generated from README.Rmd. Please edit that file -->

<!-- badges: start -->

<!-- badges: end -->

<img src="README_files/figure-gfm/showbridge-1.png" style="display: block; margin: auto;" />

<!-- ## Passionate about optimising how humans work with data -->

<!-- ## Passionate about bridging the gap between humans and their data -->

## A passion for building bridges between people and their data

  - Andrea Marion Marquard
  - 23 Feb 1986
  - Copenhagen, Denmark

I have an eye for optimisation and automation of data analysis
workflows, and I believe in the power of giving people meaningful access
to their data.

This means creating interactive data dashboards, and automating
reporting by creating eg. Word/Excel/Powerpoint products directly from
R.

To bring these solutions to life I use R package development, R
markdown, Shiny tidyverse, unit testing, github actions and bash where
needed.

I want to make life easier for both the data scientist as well as the
people who own the data.

*\[scroll to the bottom to see how the bridge above was built\]*

### Experience

`Jun 2021 - Mar 2022` **Maternity leave** (9 months)

`Mar 2021 - Present` **Director, Head of Data Science & Automation**
(Clinical Microbiomics)

  - Leading efforts to automate and scale the data analysis operations
  - Lead developer of internal R packages and Shiny dashboards
  - Managing a team of student assistants

`Aug 2019 - Mar 2021` **Bioinformatics specialist** (Clinical
Microbiomics)

  - Statistical analysis of microbiome data, and reporting results to
    clients
  - Identified a need for better code sharing, automation and
    reproducibility
  - Initiated and led development of:
      - internal R package to track and maintain analysis code
      - internal R package to automate generation of reports
      - code testing during CI, and branch protection
      - Shiny dashboards for clients to interact with their data

`Jun 2019` **Freelance bioinformatics expert** (Immumap Services)

  - Developed an NGS software tool with a Shiny interface to be used by
    lab staff

`Aug 2017 - Aug 2018` **Maternity leave** (12 months)

`Oct 2016 - May 2019` **Postdoc** (T-cells and Cancer lab, DTU Health
Technology)

  - Data analysis pipeline for single cell sequencing of T cells
  - Custom analysis pipelines for synthetic DNA barcodes
  - Pipeline to predict neoepitopes at exon-exon junctions in aberrant
    mRNA splicing
  - R package for predicting cross-reactivity of T-cells for
    immunotherapy
  - Shiny dashboard for ggplot2-powered visualisation of high-throughput
    data (available
    [here](https://marquard.shinyapps.io/barracoda-viz/))

`Jul 2015 - Apr 2016` **Maternity leave** (9 months)

`Jun 2015` **External research stay** (Ton Schumacher Lab, Antoni van
Leeuwenhoek Hospital, Amsterdam)

### Awards

`2017` Recipient of DTU’s Young Researcher Award

### Education

`2012 - 2016` **PhD in Cancer Genomics** (DTU Bioinformatics)

  - A tool for analysing NGS data from a novel screening method using
    DNA barcodes (available
    [here](https://services.healthtech.dtu.dk/service.php?Barracoda-1.8))
  - R package for machine learning-based tracing of tissue source of
    cancer cells (interface available
    [here](https://services.healthtech.dtu.dk/service.php?TumorTracer-1.1))
  - Mutation patterns in genomic data from \>5000 cancer patients reveal
    vulnerability to chemotherapy

`2012` **Machine Learning course** (by Stanford Uni at Coursera.org)

`2006 - 2012` **BSc and MSc in Molecular Biomedicine** (Uni. of
Copenhagen)

`2006` **Study abroad** (University of New South Wales, Australia)

### Volunteering & student jobs

`2007 - 2011` **Board member and chairman** (Biocenter Student House) |
*Learning to organise events, people and paperwork.*

`2010 - 2011` **Student assistant** (The Finsen Laboratory, Copenhagen
University Hospital) | *Responsible for genotyping of mice for breeding
and experiments.*

`2008 – 2010` **Student assistant** (Pediatric Oncology Research
Laboratory, Copenhagen University Hospital) | *Preparation of patient
blood samples for diagnosis and research, eg. buffy coat and DNA
extraction.*

`2007 – 2008` **Assistant press officer** (Dept. of Biology, University
of Copenhagen) | *Design of department website and English
translations.*

### Publications

| title                                                                                                                                                                              | author                                                                  | journal                                         | number                | cites | year |
| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------- | :---------------------------------------------- | :-------------------- | ----: | ---: |
| [Human endogenous retroviruses form a reservoir …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:MXK_kJrjxJIC) | SK Saini, AD Ørskov, AM Bjerregaard, A Unnikrishnan, …                  | Nature communications                           | 11 (1), 1-14          |    16 | 2020 |
| [Tumor-infiltrating T cells from clear cell rena…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:0EnyYjriUFMC) | UK Hansen, S Ramskov, AM Bjerregaard, A Borch, R Andersen, A Draghi, …  | Frontiers in immunology                         | 11, 373               |    11 | 2020 |
| [CD8+ T cells from patients with narcolepsy and …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:UebtZRa9Y70C) | NW Pedersen, A Holm, NP Kristensen, AM Bjerregaard, AK Bentzen, …       | Nature communications                           | 10 (1), 1-12          |    53 | 2019 |
| [Prediction of neoepitopes from murine sequencin…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:LkGwnXOMwfcC) | AM Bjerregaard, TK Pedersen, **AM Marquard**, SR Hadrup                 | Cancer Immunology, Immunotherapy                | 68 (1), 159-161       |     3 | 2019 |
| [T cell receptor fingerprinting enables in-depth…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:roLk4NBRz8UC) | AK Bentzen, L Such, KK Jensen, **AM Marquard**, LE Jessen, NJ Miller, … | Nature biotechnology                            | 36 (12), 1191-1196    |    67 | 2018 |
| [A meta-proteomics approach to study the intersp…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:eQOLeE2rZwMC) | J Herschend, ZBV Damholt, **AM Marquard**, B Svensson, SJ Sørensen, …   | Scientific reports                              | 7 (1), 1-13           |    33 | 2017 |
| [Quantification of within-sample genetic heterog…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:IjCSPb-OGe4C) | P Martinez, C Kimberley, NJ BirkBak, A Marquard, Z Szallasi, TA Graham  | Scientific reports                              | 7 (1), 1-12           |     4 | 2017 |
| [Large-scale detection of antigen-specific T cel…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:qjMakFHDy7sC) | AK Bentzen, **AM Marquard**, R Lyngaa, SK Saini, S Ramskov, M Donia, …  | Nature biotechnology                            | 34 (10), 1037-1045    |   212 | 2016 |
| [Sequenza: allele-specific copy number and mutat…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:u5HHmVD_uO8C) | F Favero, T Joshi, **AM Marquard**, NJ Birkbak, M Krzystanek, Q Li, …   | Annals of Oncology                              | 26 (1), 64-70         |   396 | 2015 |
| [Pan-cancer analysis of genomic scar signatures …](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:u-x6o8ySG0sC) | **AM Marquard**, AC Eklund, T Joshi, M Krzystanek, F Favero, ZC Wang, … | Biomarker research                              | 3 (1), 1-10           |   137 | 2015 |
| [TumorTracer: a method to identify the tissue of…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:9yKSN-GCB0IC) | **AM Marquard**, NJ Birkbak, CE Thomas, F Favero, M Krzystanek, …       | BMC medical genomics                            | 8 (1), 1-13           |    45 | 2015 |
| [Plasmodium falciparum expressing domain cassett…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:Tyk-4Ss8FVUC) | SS Berger, L Turner, CW Wang, JEV Petersen, M Kraft, JPA Lusingu, …     | PloS one                                        | 8 (7), e69117         |    59 | 2013 |
| [Plasmodium falciparum erythrocyte membrane prot…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:zYLM7Y9cAGgC) | T Lavstsen, L Turner, F Saguti, P Magistrado, TS Rask, JS Jespersen, …  | Proceedings of the National Academy of Sciences | 109 (26), E1791-E1800 |   264 | 2012 |
| [Evidence for in vitro and in vivo expression of…](https://scholar.google.com/citations?view_op=view_citation&hl=da&user=UFlhrz0AAAAJ&citation_for_view=UFlhrz0AAAAJ:Y0pCki6q_DkC) | CW Wang, T Lavstsen, DC Bengtsson, PA Magistrado, SS Berger, …          | Malaria journal                                 | 11 (1), 1-11          |    28 | 2012 |

### How I built the bridge

``` r
#' Plot a bridge using cosine function and ggplot lines and segments
#'
#' @param pillars_at_x the x-axis value at which to plot the pillars
#' (at `c(-pillars_at_x, pillars_at_x)`) in the graph coordinates
#' @param n_ropes the number of vertical ropes to show 
#' @param n_x the number of x values used to plot the curves (higher value
#' gives smoother curve)
#' 
#' @return a ggplot object
#' @export
#'
#' @examples
#' plot_bridge(2)
#' 
plot_bridge <- function(pillars_at_x, n_ropes = 61, n_x = 10000) {
  
  ropes_at_x <- seq(1, n_x, length.out = n_ropes) %>% round()
  
  bridge_coords <- tibble::tibble(
    # x values around two pillars at position p and minus p
    x = seq(-2 * pillars_at_x, 2 * pillars_at_x, length.out = n_x),
    # Hyperbolic cosine to get the centre curve
    y = cosh(x)
  ) %>%
    # Extend by repeating parts of the center curve at the ends
    mutate(y = case_when(x < -pillars_at_x ~ lead(y, n_x / 2),
                         x > pillars_at_x ~ lag(y, n_x / 2),
                         TRUE ~ y)
    ) 
  
  bridge_coords %>%
    ggplot(aes(x = x, y = y)) +
    geom_line(size = 1) +
    ylim(0.5, cosh(pillars_at_x)) +
    
    # add road
    geom_hline(yintercept = 1, size = 1) +
    
    # add pillars
    geom_vline(xintercept = c(-pillars_at_x, pillars_at_x), size = 1.3) +
    
    # add ropes
    geom_segment(aes(x = x, y = 1, xend = x, yend = y),
                 data = ~ .x[ropes_at_x, ]) +
    
    # remove all axes etc.
    theme_void()
}
```
